scss/civicrm/common/_forms.scss

```diff
@@ -536,7 +539,7 @@ $control-height: 30px;
     }
 
     .cke_combo_text {
-      @extend %open-sans-font-family;
+      @include open_sans_font_family();
```

Comment:
We include this mixin all over the place. This means there has to be a rule from the designer smth like "put that font here, here, here, here, here and here, but not here or here... etc". If that's the case - totally fine. If not - then may be the mixin should be used in some parent block so it covers all the children that currently utilise that mixin?

This comment is related to all places where the mixin is applied.

Reply:
I agree with the idea to keep this style in a single place. In this PR I am only converting `%open-sans-font-family` placeholder into `mixin` so that we can avoid `@extend` only. So, I didn't really take time to understand why the placeholder is used in these places. I went to the commit where this files is created(https://github.com/compucorp/org.civicrm.shoreditch/commit/2ba1801cb7bf738a4a35b48bc0f02e35fbf0b024).  It was kept like this, and i hope there is some reason in doing so. I am keeping it as it is as this investigation increases the scope of the PR.